src/main/resources/xlr/CreateAndStartSubRelease.py

```diff
@@ -9,6 +9,11 @@
 
 from xlr.XLReleaseClientUtil import XLReleaseClientUtil
 
+def find_planned_gate_task(tasks):
```

Comment:
Done, this works:
if task.getType() == "xlrelease.GateTask" and task["status"] == "PLANNED":